nvtabular/ops/stat_operator.py

```diff
@@ -29,7 +29,7 @@ class StatOperator(Operator):
     def __init__(self):
         super(StatOperator, self).__init__()
 
-    def fit(self, columns: ColumnNames, ddf: dask_cudf.DataFrame) -> Any:
+    def fit(self, columns: ColumnNames, ddf: dd) -> Any:
```

Comment:
Is 'dd' the right type here? 
```suggestion
    def fit(self, columns: ColumnNames, ddf: dd.DataFrame) -> Any:
```

Reply:
Oops